### ISSUE TITLE
[Documentation] Add accessibility features to VMenu examples

### ISSUE THREAD

**Issue by phiter** [vuetifyjs/vuetify#11390]
### Problem to solve
The current VMenu examples don't pass down attrs via the activator slot. Since most people just copy/paste stuff from there into their own codebases, adding the a11y stuff to the examples will help make sure most people are using these features.

### Proposed solution
In the VMenu examples where the activator slot is used, also pass down attrs with v-bind

**Comment by jacekkarczmarczyk** [comment]
Should be probably added also to date/time pickers

**Comment by phiter** [comment]
Yeah there probably are more components that need this. I can update the issue to expand into these other components.
